src/main/webapp/editdatafiles.xhtml

```diff
@@ -65,7 +65,17 @@
                                 <ui:param name="lockedFromEdits" value="#{EditDatafilesPage.lockedFromEdits}"/>                             
                             </ui:include>
                         </div>
-                    </ui:fragment>  
+                    </ui:fragment> 
+                    <p:blockUI block="datasetForm" widgetVar="blockDatasetForm"/>
+                    <ui:include src="file-edit-popup-fragment.xhtml">
```

Comment:
TO be clear, are you adding the file-edit-popup-fragment to this page? I ask because we hadn't yet consolidated all the different kebab options for all the pages and my concern is that just adding it here might consolidate some logic but it might also break other things (if there are duplicates). How well did you test this? Not just the functions you were working on, but all of the ones that have a popup extracted to this fragment?